Bottom: 2939ff1342df5aa2700167ed414992a239d1b907
Top:    75fc424d4ec3c1f5d7efc5466d485061c3f3b5b4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 03:48:22 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
